src/plugins/oSnap/components/Input/MethodParameter.vue

```diff
@@ -67,12 +91,25 @@ function validateBytesInput(value: string) {
   return isBytesLike(value);
 }
 
+// provide better feedback/validation messages for bytes32 inputs
 function validateBytes32Input(value: string) {
   try {
-    if (value.slice(2).length > 64) {
-      throw new Error('String too long');
+    const data = value?.slice(2) || '';
+
+    if (data.length < 64) {
+      validationErrorMessage.value = 'Value too short';
+      throw new Error('Less than 32 bytes');
+    }
+
+    if (!isBytesLike(value)) {
```

Comment:
this check should be done last.  i think it should either be in this order, too short, too long, isBytesLIke

Reply:
fixed